js/navigation.sticky.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+
+  Drupal.behaviors.megatronStickyNav = {
+    attach: function(context, settings) {
+      window.onscroll = function() { stickyNav() };
```

Comment:
Should this be `window.addEventListener('scroll', stickyNav)`?  And may be good to add some debounce/throttling because this event can be a performance suck.

https://www.html5rocks.com/en/tutorials/speed/animations/